src/util/Util.js

```diff
@@ -526,7 +526,7 @@ class Util {
    */
   static cleanContent(str, message) {
     return str
-      .replace(/@(everyone|here)/g, '@\u200b$1')
+      .replace(/@/g, '@\u200b')
       .replace(/<@!?[0-9]+>/g, input => {
```

Comment:
this regex will never be triggered as all `@`s will have a whitespace after them, perhaps add the whitespace later on?

Reply:
True. I changed it so it adds the zero width space *after* cleaning the message content.